topics/spacevim/index.md

```diff
@@ -0,0 +1,17 @@
+---
+aliases: space-vim
+created_by: Wang Shidong
+display_name: SpaceVim
+github_url: https://github.com/SpaceVim/SpaceVim
+logo: spacevim.png
+released: January 26, 2017
+short_description: SpaceVim is a community-driven vim distribution that seeks to provide layer feature.
```

Comment:
The phrasing still reads a little strangely to me here, specifically "seeks to provide layer feature". Maybe:

> SpaceVim is a community-driven distribution of the vim editor that allows managing your plugins in layers.